apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -6,42 +6,76 @@ import {
 } from "react-hook-form";
```

Comment:
Change imports here as well, since we now export those from ~/libs/types/types.ts